package.json

```diff
@@ -44,6 +44,7 @@
     "@convoform/release": "workspace:*",
     "@convoform/tsconfig": "workspace:*",
     "@release-it/conventional-changelog": "^8.0.1",
+    "cross-env": "^7.0.3",
```

Comment:
Cannot we use exist package "dotenv-cli" for this purpose?

https://github.com/entropitor/dotenv-cli#setting-variable-from-command-line

![image](https://github.com/user-attachments/assets/caac94f4-40ce-45f5-879c-00f1c9e1059d)

Reply:
No I dont think so because 
"dev": "dotenv -e .env.local -- npx dotenv-cli set NEXT_PUBLIC_PACKAGE_ENV=development turbo run dev",
will possibly work on windows but on linux it will generate error.

Reply:
cross-env provides cross platform compatiblity.